containers/kubernetes/how-to/monitor-cluster.mdx

```diff
@@ -18,27 +18,57 @@ categories:
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
   - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+  - You have [activated](/observability/cockpit/how-to/activate-cockpit/) your observability Cockpit
 </Message>
 
-## How to access the Kubernetes dashboard
+Scaleway's observability Cockpit provides automated monitoring of your Kubernetes cluster. As Kubernetes environments grow in complexity, monitoring becomes crucial for optimal performance. Visualization of the clusters' metrics are available in a hosted Grafana.
+Grafana's rich visualizations and ease of use make it an ideal choice. Cockpit offers a hassle-free solution, providing real-time insights without extra infrastructure.
 
-1. Click **Kubernetes** in the **Containers** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Kubernetes Kapsule dashboard displays.
-2. Click the name of the cluster you want to monitor. The cluster overview page displays.  This page provides several pieces of information about your cluster:
-    * Cluster information
-    * Auto Upgrade
-    * Cluster tags
-    * Renewal of the kubeconfig file to restore access permissions
-    * Download of the kubeconfig file
-    * The option to delete the cluster
+[Cockpit](/observability/cockpit/quickstart/) empowers you to monitor the clusters' control plane, nodes, managed resources and the cluster system applications.
+* Detect incidents and diagnose failures in the earliest stages
+* Get new insights about control plane and worker nodes usage such as CPU, memory, to optimize your resource allocation
+* Achieve a better understanding of your infrastructure by analyzing data on your API server, system applications, up and down scaling behavior
 
-If you click the **Dashboard** button, you are taken to your cluster's Kubernetes dashboard. Here you can troubleshoot your containerized application, manage the cluster resources, get an overview of applications running on your cluster, and create or modify certain resources.
+## How to access your Cockpit Grafana dashboard
 
-If you click the **Pools** tab, you are able to add, edit or delete a pool on your cluster.
+<Message type="important">
+  You must have [activated](/observability/cockpit/how-to/activate-cockpit/) your Cockpit. Cockpit’s free plan retention time being 31 days for metrics and 7 days for logs.
+</Message>
+
+1. Open your Cockpit in the [Scaleway console](https://console.scaleway.com/cockpit/overview).
```

Comment:
There should be a step after this one to create a Grafana user, so they are able to log in to Grafana with their credentials or maybe add it to the requirements